code/src/components/TaskCounter.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+/* Counter component, it shows how many todos i completed, and how many are left.
+we filter the tasks depening on if they are completed or not */
+
+const TaskCounter = () => {
+  const allTasks = useSelector((store) => store.tasks.items)
+
+  const completedTasks = allTasks.filter((singleTask) => singleTask.complete)
+  const tasksLeft = allTasks.filter((singleTask) => !singleTask.complete)
```

Comment:
Very nice solution! Cool that you have both a counter as well as how many todos are left!